src/signals/incident/components/form/MapSelectors/Asset/Selector_v2_removeafterfinishepic5440/DetailPanel/DetailPanel.tsx

```diff
@@ -40,76 +32,20 @@ export interface DetailPanelProps {
 const DetailPanel: FC<DetailPanelProps> = ({ language }) => {
   const shouldRenderMobileVersion = useMediaQuery({
     // Set breakpoint to mobile instead of tablet since desktop will get the mobile version when zoom on 200% which is not accesible with keyboard.
-    query: breakpoint('max-width', 'mobileL')({ theme: ascDefaultTheme }),
+    query: breakpoint('max-width', 'tabletM')({ theme: ascDefaultTheme }),
   })
 
   const dispatch = useDispatch()
-  const { address, selection, removeItem, setItem, setLocation, meta } =
+  const { address, selection, removeItem, setLocation, meta } =
     useContext(AssetSelectContext)
   const { featureTypes } = meta
   const featureStatusTypes = meta.featureStatusTypes || []
 
   const addressValue = address ? formatAddress(address) : ''
 
-  const unregisteredAsset =
-    selection && selectionIsUndetermined(selection[0])
-      ? selection[0]
-      : undefined
-
   const selectionOnMap =
     selection && selectionIsObject(selection[0]) ? selection : undefined
 
-  const [showObjectIdInput, setShowObjectIdInput] = useState(
-    selection && selection[0].type === UNKNOWN_TYPE
-  )
```

Comment:
this UKNOWN_TYPE is used in other places. But since we're removing it here were it's set. Should all related code not be removed? Or do we keep it since older incidents make use of it?

Reply:
Ahh UKNOWN_TYPE still lives in the current version. We can remove UNKNOWN_TYPE when ditching Selector for v2.
<img width="437" alt="Screenshot 2023-11-01 at 15 54 32" src="https://github.com/Amsterdam/signals-frontend/assets/7532781/055da588-ba52-4188-8e64-bd57d36121bb">